utils/sparse.py

```diff
@@ -308,6 +309,8 @@ def save_sample_inputs_outputs(
                 file_idx = f"{exported_samples}".zfill(4)
 
                 sample_input_filename = os.path.join(f"{sample_in_dir}", f"inp-{file_idx}.npz")
+                if save_inputs_as_uint8:
+                    sample_in = (255 * sample_in).astype(numpy.uint8)
```

Comment:
```suggestion
                    sample_in = (255 * sample_in).to(dtype=torch.uint8)
```

Reply:
sample_in is a torch tensor at this point in the code. astype throws an error. Torch type conversion works fine. Verified that after this change it fixes the issue.